apps/spruce/src/components/VersionRestartModal/VersionTasks.tsx

```diff
@@ -1,3 +1,4 @@
+import styled from "@emotion/styled";
 import { Divider } from "components/styles/divider";
```

Comment:
Can we prevent the scroll-within-scroll behavior?

https://github.com/evergreen-ci/ui/assets/47064971/68d599f2-641d-4e5b-945f-21f5a391864e

Reply:
Give it a shot now!